src/software/world/robot.h

```diff
@@ -174,6 +174,27 @@ class Robot
      */
     AngularVelocity angularVelocity() const;
 
+    /**
+     * Returns if the ball is in the mouth of the robot, i.e. near the dribbler
+     *
+     * @return ball in mouth
+     */
+    bool ballInMouth() const;
```

Comment:
😬 this is the most clear, but can't tell `ballInMouth` is going to be memed

Reply:
Oh it absolutely will be :trollface: